addons/portal/views/portal_templates.xml

```diff
@@ -86,7 +86,8 @@
     <template id="portal_breadcrumbs" name="Portal Breadcrumbs">
         <ol t-if="page_name != 'home'" class="o_portal_submenu breadcrumb mb-0 py-2 flex-grow-1 row">
             <li class="breadcrumb-item ml-1"><a href="/my/home" aria-label="Home" title="Home"><i class="fa fa-home"/></a></li>
-            <li t-if="page_name == 'my_details'" class="breadcrumb-item">Details</li>
+            <li t-if="page_name == 'my_details'" class="breadcrumb-item active"><t t-esc="page_title"/></li>
+            <li t-if="page_name == 'security'" class="breadcrumb-item active"><t t-esc="page_title"/></li>
```

Comment:
It's because it becomes more complicated in the commit where we update /my/account (revamp address), but it's true that I can use your concise format whenever possible